terraform/modules/networking/ci_security_group.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_security_group" "aws_interface_endpoints" {
+  name   = "${local.vpc_name}-aws-interface-endpoints"
+  vpc_id = module.vpc.vpc_id
+  ingress {
+    from_port   = 0
+    protocol    = "-1"
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
Are we allowing all ipv4 addresses because `private_dns_enabled` is enabled & it's easier this way?

Reply:
Thinking about it, I believe it is because of `private_dns_enabled`. I can also see similar configuration here - https://github.com/hmrc/platsec-ci-terraform/blob/main/modules/networking/ci_security_group.tf